src/bootstrap/src/core/build_steps/test.rs

```diff
@@ -1769,7 +1769,10 @@ NOTE: if you're sure you want to do this, please open an issue as to why. In the
         cmd.arg("--host").arg(&*compiler.host.triple);
         cmd.arg("--llvm-filecheck").arg(builder.llvm_filecheck(builder.config.host_target));
 
-        if let Some(codegen_backend) = builder.config.default_codegen_backend(compiler.host) {
+        if let Some(codegen_backend) = builder.config.codegen_backends(compiler.host).first() {
```

Comment:
Since we don't seem to run commands for each registered backend, I simply look at the first for now. Should I add a FIXME comment on it?

Reply:
`default_codegen_backend()` is literally implemented as `codegen_backends().first()`.

Reply:
I just submitted it now :laughing: It was a pending comment, but GH uses the original time when it was written, not when it was submitted, which is a bit confusing.